tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -3,12 +3,10 @@ set -e
 while getopts p:d: parameter_Option
 do case "${parameter_Option}"
 in
-p) PYTHON_VER=${OPTARG};;
```

Comment:
Removing python version as input is causing some pipelines to fail. For example the orttraining-distributed pipeline is not able to find the python executable and failing as a result.

Reply:
Fixed in #8136